cli/src/config.rs

```diff
@@ -395,9 +395,18 @@ impl ConfigEnv {
     /// Resolves conditional scopes within the current environment. Returns new
     /// resolved config.
     pub fn resolve_config(&self, config: &RawConfig) -> Result<StackedConfig, ConfigGetError> {
+        self.resolve_config_with_command(config, None)
+    }
+
+    pub fn resolve_config_with_command(
```

Comment:
set the command name in `ConfigEnv`, and do `resolve_config()`. `config_env` can be reused later in order to resolve settings for cloned/inited workspace.